hooks/pre/30-upgrade.rb

```diff
@@ -164,6 +171,7 @@ def fail_and_exit(message)
     upgrade_step :mark_qpid_cert_for_update
     upgrade_step :migrate_candlepin, :run_always => true
     upgrade_step :remove_gutterball
+    upgrade_step :delete_event_queue
     upgrade_step :start_tomcat, :run_always => true
```

Comment:
@stbenjam We want puppet with the other pr's to create the event queue and exchange with the new filters that @barnabycourt and @jlsherrill found while debugging a downstream case. @barnabycourt is also making a change on the Candlepin side of things to help Katello out but this is the action that was deemed needed on the Katello side of things.